pages/conduct.md

```diff
@@ -0,0 +1,40 @@
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the jQuery Foundation and its projects. This includes IRC, the mailing lists, the issue tracker, jQuery events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing conduct@jquery.com. For more details please see our [Reporting Guidelines](https://jquery.org/conduct/reporting/)
+
+* **Be friendly and patient.**
+
+* **Be welcoming.** We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+* **Be considerate.** Your work will be used by other people, and you in turn will depend on the work of others. Any decision you take will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we're a world-wide community, so you might not be communicating in someone else's primary language.
+
+* **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a productive one. Members of the jQuery community should be respectful when dealing with other members as well as with people outside the jQuery community.
+
+* **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
+	* Violent threats or language directed against another person.
```

Comment:
Wow, this is a tough thing. I'm a bit of a drive-by in this comment (though my opinion was asked privately), and this is super complicated.

Self harm is a complicated topic on its own; a symptom of other problems; a deeply misunderstood behavioral pattern that has some complicated stigma and ties into notions of humans and how they relate to having control over their own lives.. Where it intersects with online communication gets into a whole other bag of tricks: it can be a very effective way to manipulate. However, rarely one that is used purely for manipulation, and the potential for harm when dismissed as manipulation is great.

It's also deeply upsetting to a lot of people to have running across the middle of a community thread. Talk of self-harm does generally require great care, and in the middle of most technical organization's talk is not the place for emotional, supportive, nuanced discussion without derailing the topic it branches from.

I think for the purposes of this document, talk of self-harm should be not allowed.

However. The response that comes to such talk if and when it happens should be kind, caring, and supportive -- while a technical community is unlikely to have sophisticated resources for helping someone in a situation where they are harming themselves, at least a minimum of preparedness is needed to deal with it effectively. Move the conversation aside; if not private, then at least out of the line of fire of the technical discussion; Seek to connect the person who's threatening self-harm to resources that can support them in healing. However, to keep the community functioning, such talk must be strictly able to be opted out of for all participating. This means careful choice of communication channels. The enforcement documents may play a huge role in how this plays out, but they will have to be public, because of the galvanizing nature of such talk.